meinberlin/apps/votes/views.py

```diff
@@ -26,26 +27,15 @@ class ExportTokenDashboardView(
     permission_required = "a4projects.change_project"
     template_name = "meinberlin_votes/token_export_dashboard.html"
 
-    def _get_number_of_tokens(self):
-        """Return total and downloadable token count for module."""
-        return VotingToken.objects.filter(module=self.module, is_active=True).count()
```

Comment:
by removing this and using the size we save in the package, codes which have `is_active=False` will still be included in the size (but won't be included in the download). Is that something we need to fix? It would require more db queries, which is why I didn't do it for now

Reply:
I think it is fine like that. We want to use the `is_active` to deactivate tokens that might have been meddled with, right? So, that's not important for the download, as this normally happens before a token is changed to `is_active=False`.